src/test/kotlin/org/rucca/cheese/api/TaskTest.kt

```
@@ -498,79 +203,95 @@ constructor(private val mockMvc: MockMvc, private val userCreatorService: UserCr
+                deadline = taskDeadline,
+                defaultDeadline = taskDefaultDeadline,
+                resubmittable = true,
+                editable = true,
+                intro = taskIntro,
+                description = taskDescription,
+                submissionSchema = taskSubmissionSchema,
+                spaceId = spaceId,
+                categoryId = archivedCategoryId,
+            )
+            assert(false) { "Should have failed to create task in archived category" }
+        } catch (e: AssertionError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)